src/osemgrep/language_server/requests/Initialize_request.ml

```diff
@@ -67,6 +67,8 @@ let initialize_server server
           "intellij"
     | _ -> false
   in
+  Lwt_preemptive.init 1 user_settings.jobs (fun msg ->
```

Comment:
Maybe a small comment here like (* we're using lwt cooperative threads in most of the code of the language server,
but we still need sometimes to execute code using Unix. or Common. that access files in a blocking way. However
we can wrap those calls in the Lwt_preemptive library which internally use Thread to at least not block
the whole lwt event loop but just block threads in a pool. This init calls creates such a thread pool internally.

Reply:
Will do